src/libraries/System.Numerics.Colors/src/System.Numerics.Colors.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Was talking with @JeremyKuhne and his expectation matched mine, which was this was going to be part of the existing `System.Numerics.Vectors` package rather than a net new package.

Particularly for acceleration and support around `float`, there is likely to be a need for these to be used in conjuction. The "downside" is that since `System.Numerics.Vectors` is in-box and frozen downlevel these new types would never be available on .NET Framework. However, that isn't a concern from the WinForms/WPF team (much as any of the other new graphics related vector/matrix/plane/etc APIs aren't available downlevel).

CC. @ericstj, just for additional confirmation there isn't something we're missing here

Reply:
Yeah, that sounds reasonable.  Also this project was only targeting NetCoreAppCurrent and not a package - so not really a change in targeting - but we should avoid the new assembly.  If folks decide later that there needs to be a polyfill then that could be created, but we don't proactively do that.

Reply:
@AlexRadch could you update the location to be in S.N.Vectors as indicated above? After that I can complete the review and ideally get this merged.